.eslintrc.js

```diff
@@ -181,6 +181,8 @@ module.exports = {
 				};
 			} ),
 		} ],
+		// TODO: removeme, maybe? Without this linting fails horribly on Windows, many thousands of false positives :)
+		// 'linebreak-style': 'off',
```

Comment:
Is this supposed to be part of this PR?

Reply:
Not really. Still work-in-progress though, will do the `TODO` :)